### DISCUSSION TITLE
.aab build failed

### DISCUSSION THREAD

**Discussion by sesisoft-worldflipper** [airsdk/Adobe-Runtime-Support#974] [Q&A]
![fail1](https://user-images.githubusercontent.com/77314595/125282753-b21cfe00-e352-11eb-89d8-4e6847556592.jpg)
![fail2](https://user-images.githubusercontent.com/77314595/125282767-b5b08500-e352-11eb-8a78-48a3c4bedc92.jpg)

I am working with mobile game project.
It is build as .apk and .ipa very well.
But -target aab option is not working.
It has many sub ANEs.
Please help me.

**Comment by ajwfrost** [comment]
Hi

It's a little tricky to see what's going on but for debugging things like this, I would suggest editing your "adt.cfg" file within the AIR SDK 'lib' folder, and setting it up to have `KeepAndroidStudioOutput=true`

Then if you try building again with -target aab, this time it will create a subfolder for you called "AndroidStudioProject" which you can open within Android Studio and do a "Build Project" command.

You can then look at the XML file that's reporting this issue, and hopefully work out where those resource requests are coming from, and where the files should be. It may just be a missing file extension within one of the resources that perhaps is packaged up in an ANE or should have been provided to the list of input files for ADT, etc: but it would be a lot easier to find and fix I think from Android Studio.

thanks
